crates/loader/src/assets.rs

```diff
@@ -103,6 +104,24 @@ pub(crate) async fn change_file_permission(
     Ok(())
 }
 
+pub fn validate_allowed_http_hosts(http_hosts: &Option<Vec<String>>) -> Result<()> {
```

Comment:
nit: I don't think this belongs in the `assets` module; if there's no existing home for it then maybe create a new `validation` module?

Reply:
Yeah, add a `validation` module for it. Thank you.